pkg/grpc/authenticator.go

```diff
@@ -46,7 +46,9 @@ func NewAuthenticatorFromConfiguration(policy *configuration.AuthenticationPolic
 		}
 		return NewTLSClientCertificateAuthenticator(
 			clientCAs,
-			clock.SystemClock), nil
+			clock.SystemClock,
```

Comment:
add roughtime support

Reply:
The main goal of the Clock interface right now is just to stub out the clock as part of unit tests. Well, there's also the I/O time compensating clock we use in bb_worker to compensate for delays caused by FUSE, but that's just a detail.

I think that stuff like roughtime is best implemented by the operating system itself, through some timekeeping daemon, right? Pretty exciting stuff. I hadn't heard of it before. Thanks for sharing!